src/widgets/scroll.rs

```diff
@@ -22,9 +22,9 @@ use crate::ConstraintLimit;
 #[derive(Debug)]
 pub struct Scroll {
     contents: WidgetRef,
-    content_size: Size<Px>,
-    control_size: Size<Px>,
-    scroll: Dynamic<Point<Px>>,
+    content_size: Dynamic<Size<Px>>,
+    control_size: Dynamic<Size<Px>>,
+    pub scroll: Dynamic<Point<Px>>,
```

Comment:
Running cargo clippy should produce a warning for missing documentation on this field.